tests/console/apache.pm

```diff
@@ -133,9 +134,9 @@ sub run {
             assert_script_run q{sed -ie 's/^Group daemon$/Group www/' /tmp/prefork/httpd.conf};
 
             # Run and test this new environment
-            assert_script_run 'httpd2-prefork -f /tmp/prefork/httpd.conf';
-            assert_script_run 'until ps aux|grep wwwrun|grep -E httpd\(2\)?-prefork; do echo waiting for httpd2-prefork pid; done';
-            assert_script_run 'ps aux | grep "\-f /tmp/prefork/httpd.conf" | grep httpd2-prefork';
+            assert_script_run 'httpd-prefork -f /tmp/prefork/httpd.conf';
```

Comment:
I am not sure if older sle version like sp5/sp4 is using `httpd-prefork` as well, can you please double confirm?

Reply:
I checked in a VM. All versions including 12-SP5 use `httpd-prefork`. AFAICS `httpd2-prefork` and `httpd-prefork` both are available.